provision/acc_provision/templates/provision-config.yaml

```diff
@@ -71,6 +71,7 @@ registry:
 #kube_config:
   # ovs_memory_limit: "20Gi"            # override if needed, default is "1Gi"
   # reboot_opflex_with_ovs: "false"     # override if needed, default is "true"
+  # ep_registry: "k8s"                  # default is null, set to "k8s" to enable ERSPAN feature
```

Comment:
the quotes are not required here (arguably the earlier examples are also not accurate)